Content/SCPCB/Menus/ConsoleMenu.as

```diff
@@ -0,0 +1,316 @@
+
+
+class menu_Console : GUI {
+	GUI@ canvas;
+
+	GUI@ canvasMain;
+	GUI@ canvasDeveloper;
+	GUI@ canvasCheats;
+	GUI@ canvasOptions;
+
+	GUIScrollPanel@ scrollTest;
+
+	menu_Console() { super("ConsoleMenu");
+
+	align=Alignment::Center;
+	size=Vector2f(70,70);
+	visible=false;
+
+	GUILabel@ header=GUILabel(@this);
+	header.text="CONSOLE MENU";
+	header.align=Alignment::Top;
+	header.height=GUI::Resolution.y*0.07;
+	header.fontScale=3;
+	header.margin={4,GUI::Resolution.y*0.035,4,1};
+
+	@canvas=GUI(@this);
+	canvas.align=Alignment::Fill;
+	canvas.margin={GUI::Resolution.x*0.07,GUI::Resolution.y*0.01,2,0};
+
+
+	// Main section ----
+
+	@canvasMain=GUI(@canvas);
+	canvasMain.align=Alignment::Fill;
+
+	menu_Console_TextPanel@ textPanel = menu_Console_TextPanel(@canvasMain);
+	textPanel.align=Alignment::Top;
+	textPanel.height=8;
+	textPanel.margin={1,0,1,0.5};
+
+	@ConsoleMenu_ButtonPanel=menu_Console_ButtonPanel(@canvasMain);
```

Comment:
Using namespaces is the appropriate solution over using snake case here

Reply:
Sounds good, I'll go ahead with this cleanup.
Question to be asked; should the GUI elements all go in the GUI namespace too? They're currently all prefixed with GUI* anyway.

Reply:
The pausemenu has been refactored into the final format. the other menus will follow suit